tests/docstrings/T05.icry.stdout

```diff
@@ -12,4 +12,8 @@ Checking module T05
       Using exhaustive testing.
       Testing... Passed 1 tests.
       Q.E.D.
-Successes: 2, No fences: 1, Failures: 0
+    :exhaust p
+      Using exhaustive testing.
+      Testing... Passed 1 tests.
+      Q.E.D.
```

Comment:
This example already has an explicit `:exhaust` pragma in the docstrings:

https://github.com/GaloisInc/cryptol/blob/86ee4b688c2e55493ae8cbb13979666654e97d0f/tests/docstrings/T05.md?plain=1#L13-L15

As such, this runs `:exhaust p` twice (you can see the output of the other `:exhaust p` invocation above). Is it possible to detect examples like this to avoid redundantly running `:exhaust` again?

Reply:
Good idea. I can poke around at the code consuming the gathered-up docstrings / see about de-duplicating what's extracted there, rather than trying to detect this based on the docstring text itself.

Reply:
Oh sure! If that's an acceptable adjustment to what we originally wanted, it's definitely an easy way to avoid duplicates without any kind of janky cleanup after gathering up / extracting all the code blocks.

I'll adjust what I had queued up re: changes to the reference manual and implement this check instead.

Reply:
Actually -- We don't know about the structure of the docstring code blocks until the extraction is done, so we'll have to move some of this into the `REPL.Command` module.

I was hoping to just replace `<>` with `<|>` where I'm adding the new doctring test, but this is too aggressive, as it'll skip adding the `:exhaust` if there are _any_ docstring comments, even if there are no other code fences. That would have been cool :)

Reply:
6af7930d